lib/Doctrine/ORM/EntityRepository.php

```diff
@@ -58,8 +58,8 @@ class EntityRepository implements ObjectRepository, Selectable
     /**
      * Initializes a new <tt>EntityRepository</tt>.
      *
-     * @param EntityManager         $em    The EntityManager to use.
-     * @param Mapping\ClassMetadata $class The class descriptor.
+     * @param EntityManagerInterface    $em    The EntityManager to use.
+     * @param Mapping\ClassMetadata     $class The class descriptor.
```

Comment:
Can be dropped completely (comment is not really telling anything useful)

Reply:
You mean fix to that:
```
    /**
     * Initializes a new <tt>EntityRepository</tt>.
     *
     * @param EntityManagerInterface $em
     * @param Mapping\ClassMetadata  $class
     */
```

Reply:
@koftikes nope, drop both `@param`. They aren't needed since types are declared properly.

Reply:
@lcobucci nope it's not correct params declarations need for normal code 
![image](https://user-images.githubusercontent.com/8553840/49872805-41747480-fe2b-11e8-8898-b2217e38ccf7.png)

Reply:
That's an IDE warning that we ignore: our coding standards enforce dropping any docblock entry that doesn't describe the actual purpose of the referenced symbol.

Reply:
really? you have all descriptions on each constructor. lol.